myst_nbs/howto/Model_Builder.myst.md

```diff
@@ -0,0 +1,258 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: pymc
+  language: python
+  name: pymc
+---
+
+# Using ModelBuilder class for deploying PyMC models 
+:::post: Sep 12, 2022:tags: model builder class, model class, linear model:category: Beginner:author: Shashank Kirtania, Thomas Wiecki:::
```

Comment:
This need to have every element in a different line and use the colon and pointy parenthesis syntax as shown in https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell. The target above the title so we can reference this notebook from other pages is also missing.

See how the preview for this notebook https://pymcio--419.org.readthedocs.build/projects/examples/en/419/howto/Model_Builder.html does not have any highlighted icon nor text in the left sidebar, nor it has the github/colab/binder badges nor the tags listed on the right sidebar like other notebook (that have been updated) do. You can use https://pymcio--419.org.readthedocs.build/projects/examples/en/419/causal_inference/regression_discontinuity.html as an comparison reference. This is because sphinx doesn't understand the post directive and therefore doesn't have the metadata for this.